engine/src/main/java/org/terasology/world/generator/WorldGenerator.java

```diff
@@ -24,24 +24,56 @@
 import org.terasology.world.generation.EntityBuffer;
 import org.terasology.world.generation.World;
 
+/**
+ * World generator is an interface responsible for generating worlds from their seed
+ */
 public interface WorldGenerator {
+    /**
+     * @return Uri of the current world generator instance
+     */
     SimpleUri getUri();
 
+    /**
+     * @return Seed used for creating this world generator
+     */
     String getWorldSeed();
 
+    /**
+     * Sets the seed to use for creating of the world made by this world generator.
+     * <p>
+     * NOTE: this is a String value. The long value used in the most common implementation of this interface
```

Comment:
Instead of writing "in the most common implementation" link to the implementation that you're talking about.